src/session.js

```diff
@@ -115,6 +119,11 @@ ghostdriver.Session = function(desiredCapabilities) {
     _capsPageSettingsPref = "phantomjs.page.settings.",
     _capsPageCustomHeadersPref = "phantomjs.page.customHeaders.",
     _capsPageZoomFactor = "phantomjs.page.zoomFactor",
+    _capsPageBlacklistPref = "phantomjs.page.blacklist",
+    _capsPageWhitelistPref = "phantomjs.page.whitelist",
+    _capsUnhandledPromptBehavior = "unhandledPromptBehavior",
```

Comment:
I think those behaviours are non standard: this needs to be documented, or people are going to flood with questions on how to use it.